msprime/cli.py

```diff
@@ -161,6 +167,35 @@ def __init__(
         self._mutation_generator = msprime.MutationGenerator(
             self._random_generator, self._mutation_rate)
 
+    def hotspots_to_recomb_map(self, hotspots, recomb_rate, num_loci):
```

Comment:
We're not using ``self`` here at all, so this is more naturally a function I think. The function should have a docstring defining the input and outout. Then we can add a few unit tests in the test_cli module
```python
class TestHotspotsToRecombMap(..)
     def test_one_hotspot(...)

```
Just a few different tests to make sure we're getting all the corner cases in for this function.